.github/CHANGELOG.md

```diff
@@ -45,28 +47,28 @@
  Note that each MPI process will return the overall result of expectation value and sample generation. However, `probability` will 
  return local probability results. Users should be responsible to collect probability results across the MPI processes.
  
- The workflow for collecting probability results across the MPI processes:
+ The workflow for collecting probability results across the MPI processes is as follows:
  ```python
  from mpi4py import MPI
  import pennylane as qml
  import numpy as np
 
  comm = MPI.COMM_WORLD
  rank = comm.Get_rank()
- numQubits = 8
- dev = qml.device('lightning.gpu', wires=numQubits, mpi=True)
+ dev = qml.device('lightning.gpu', wires=8, mpi=True)
+ prob_wires = [0, 1]
 
  @qml.qnode(dev)
  def mpi_circuit():
    qml.Hadamard(wires=1)
-   return qml.probs(wires=[0, 1])
+   return qml.probs(wires=prob_wires)
 
- local_probs = mpi_circuit
+ local_probs = mpi_circuit()
  
  #For data collection across MPI processes.
  recv_counts = comm.gather(len(local_probs),root=0)
  if rank == 0:
-    probs = np.zeros(1<<numQubits)
+    probs = np.zeros(2**len(prob_wires))
  else:
     probs = None
```

Comment:
Can we use 4 spaces for indenting the Python code here?

Reply:
Done!